diff --unified --recursive --text --color package.orig/src/modules/partition/core/DeviceList.cpp package.new/src/modules/partition/core/DeviceList.cpp
--- package.orig/src/modules/partition/core/DeviceList.cpp	2020-03-28 10:43:31.000000000 +0200
+++ package.new/src/modules/partition/core/DeviceList.cpp	2020-03-31 13:20:59.721262802 +0300
@@ -118,7 +118,7 @@
 
     CoreBackend* backend = CoreBackendManager::self()->backend();
 #if defined( WITH_KPMCORE4API )
-    DeviceList devices = backend->scanDevices( /* not includeReadOnly, not includeLoopback */ ScanFlag( 0 ) );
+    DeviceList devices = backend->scanDevices( /* not includeReadOnly, not includeLoopback */ ScanFlag( 2 ) );
 #else
     DeviceList devices = backend->scanDevices( /* excludeReadOnly */ true );
 #endif
